src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -16,24 +16,66 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo()
+        {
+            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
+
+            DirectorySecurity directorySecurity = FileSystemAclExtensions.GetAccessControl(directoryInfo);
+
+            Assert.NotNull(directorySecurity);
```

Comment:
>any suggestion on how to get AccessControlSections.Access, AccessControlSections.Owner and AccessControlSections.Group out of the DirectorySecurity object? Thanks, Michael

There isn't a way, afaik. Except reflection but we don't want that. 

What about `AccessRightType` as Steve mentioned, is that modified by the AccessControlSections passed?